vital/api/link.py

```diff
@@ -12,7 +12,7 @@ def create(self, user_id: str, provider: Optional[str] = None) -> Mapping[str, s
         :param str user_id: user's id returned by service.
         """
         return self.client.post(
-            "/link/token/", {"user_id": user_id, "provider": provider}
+            "/link/token", {"user_id": user_id, "provider": provider}
```

Comment:
Does this cause any issues for fastapi?

Reply:
I'm not sure I follow? The reason this wasn't an issue was because in app-engine you don't have to be exact with URLs

Reply:
With cloud run you do